examples/expression-oriented-style/expression_oriented_style.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina/io;
+
+// You can define the return value expression using the `=>` notation instead of using
+// curly braces to define the function body block when the function body is just
+// a return statement with an expression.
+function inc1(int x) returns int => x + 1;
+
+// `inc2(int x)` is the same as` inc1(int x)`.
+function inc2(int x) returns int {
+    return x + 1;
+}
+
+var obj = object {
+    private int x = 1;
+    // Same also applies to methods
+    function getX() returns int => self.x;
+};
+
+// This can also be used with query expressions
```

Comment:
A new user might be confused with this comment. ( what can be used with query expression? The allow function, let expression or something else?)